dist/index.html

```diff
@@ -121,6 +122,33 @@ <h2>Total Cost</h2>
       </section>
       <section class="recipe hidden" id="recipe">
       </section>
+    </section>
+    <section class="pantry hidden" id="pantry-view">
+      <section class="pantryContents">
+        <div class="dropDropMenu">
+          <label for="ingredients">Add Ingredient to Pantry: </label>
+            <select name="ingredients" id="ingredient-drop-down-menu" class="selectMenu">
+            </select>
+            <form class="quantity">
+              <input type="text" placeholder="quantity" class="quantityInput" id="quantity-input">
+            </form>
+            <button type="button" class="addButton" id="add-button">Add to Pantry</button>
+        </div>
+        <div class="pantryItems">
```

Comment:
These divs could be reassessed during accessibility audit. We could utilize section, table, form, menu, header, etc. Just a note for the future, not important now